src/utils/components/SSHAccess/components/SSHCommand.tsx

```diff
@@ -38,6 +41,8 @@ const SSHCommand: React.FC<SSHCommandProps> = ({
   const [loading, setLoading] = useState(false);
   const [error, setError] = useState<Error>();
 
+  const optimisticCheckOnLoading = loading ? !sshService : Boolean(sshService);
```

Comment:
optimistic follow the optimistic frontend approach means that you take for granted that the backend will respond positively to the frontend request. And if there is any error, it will come back to the previous state. So this means that during loading we assume that if the `sshService` is undefined, will be defined after the request (because the request is for the creation of the service) and if is defined, will be undefined (because the request is for the deletion).   

It's okay for me to remove this part as maybe this approach is not widespread

Reply:
Ah, you mean this kind of approach, that's fine. Maybe shorter variable name + some comment would be the best, IMO ;)

Reply:
I removed it. It's fine. Don't want to add unnecessary code that needs comments